mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -261,6 +262,34 @@ export class CompareRunView extends Component {
         <tbody>{dataRows}</tbody>
       </table>
     );
+  }  
+
+  renderTagTable(colWidth) {
+    const dataRows = this.renderDataRows(
+      this.props.tagLists,
+      colWidth,
+      this.state.onlyShowParamDiff,
+      true,
+    );
+    if (dataRows.length === 0) {
+      return (
+        <h2>
+          <FormattedMessage
+            defaultMessage='No tags to display.'
+            description='Text shown when there are no tags to display'
+          />
+        </h2>
+      );
+    }
+    return (
+      <table
+        className='table compare-table compare-run-table'
+        style={{ maxHeight: '500px' }}
```

Comment:
```suggestion
        css={{ maxHeight: '500px' }}
```
```suggestion
        style={{ maxHeight: '500px' }}
```

Reply:
`css` is also used in other places in this file. Can I fix all in the same change?

Reply:
Yes!

Reply:
Done, fixed all of them.